src/main/java/com/workduo/configuration/querydsl/CustomMySQL8InnoDBDialect.java

```diff
@@ -0,0 +1,15 @@
+package com.workduo.configuration.querydsl;
+
+import org.hibernate.dialect.MySQL8Dialect;
+import org.hibernate.dialect.function.StandardSQLFunction;
+import org.hibernate.type.StandardBasicTypes;
+
+public class CustomMySQL8InnoDBDialect extends MySQL8Dialect {
+
+    public CustomMySQL8InnoDBDialect() {
+        super();
```

Comment:
super() 없으면 다른 sql 함수는 사용 못하나요 ?

Reply:
가능은 한데 기존 부모가 가지고 있는 기본 함수들을 못 쓸거 같은데요?!

Reply:
상속을 타고 타고 들어가니까 최상위 부모는 Dialect이 클래스인데
public abstract class Dialect implements ConversionContext {
  // ... 생략
  private final Map<String, SQLFunction> sqlFunctions = new HashMap<>();
  protected void registerFunction(String name, SQLFunction function) {
    // HHH-7721: SQLFunctionRegistry expects all lowercase.  Enforce,
    // just in case a user's customer dialect uses mixed cases.
    sqlFunctions.put( name.toLowerCase( Locale.ROOT ), function );
  }
 // ..생략
}
이 클래스 안에 함수를 저장하는 메소드랑 프로퍼티가 있네요 
그리고 MySQL8Dialect, MYSQL57Dialect 등 Dialect을 상속 받은 클래스들은 기본 생성자 안에서 
registerFunction(String name, SQLFunction function)이 메소드를 사용해서 기본 적인 함수를 저장해주는데 만약 super를 안 쓰면
부모들이 기본적으로 저장하는 함수는 못 쓰고 제가 등록한 함수만 쓸 수 있을거 같네여